ui/app/AppLayouts/Chat/controls/community/InlineNetworksComboBox.qml

```diff
@@ -0,0 +1,154 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQml 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+import SortFilterProxyModel 0.2
+
+StatusComboBox {
+    id: root
+
+    type: StatusComboBox.Type.Secondary
+    size: StatusComboBox.Size.Small
+
+    height: 48
+
+    control.enabled: !d.oneItem
+    control.padding: d.padding
+    control.spacing: d.padding
+    control.popup.y: 0
+    control.popup.padding: 0
+    control.popup.verticalPadding: 0
+    control.textRole: "name"
+    control.currentIndex: 0
+    control.indicator.visible: !d.oneItem
+
+    control.background: Rectangle {
+        radius: d.radius
+        color: "transparent"
+        border.color: Theme.palette.directColor7
+        visible: !d.oneItem
+    }
+
+    QtObject {
+        id: d
+
+        readonly property bool oneItem: control.count === 1
+
+        readonly property int padding: 8
+        readonly property int radius: 8
+        readonly property int fontSize: 13
+        readonly property int iconSize: 32
+    }
+
+    component CustomText: StatusBaseText {
+        color: Theme.palette.baseColor1
+        font.pixelSize: d.fontSize
+        font.weight: Font.Medium
+        elide: Text.ElideRight
+    }
+
+    component DelegateItem: StatusItemDelegate {
+        property alias title: titleText.text
+        property alias amount: amountText.text
+        property alias iconSource: icon.source
+        property alias cursorShape: mouseArea.cursorShape
+
+        padding: 0
+
+        background: Rectangle {
```

Comment:
The `background` code is exactly the same, except the `radius` property; I think it would be nice to add it to `StatusItemDelegate` and not duplicate the code here

Reply:
good idea!